src/helpers/files.js

```diff
@@ -139,14 +139,48 @@ const registerFileActionFallback = () => {
 
 }
 
+const newRichWorkspaceFileMenuPlugin = {
+	attach(menu) {
+		const fileList = menu.fileList
+		// only attach to main file list, public view is not supported yet
+		if (fileList.id !== 'files' && fileList.id !== 'files.public') {
+			return
+		}
+
+		// register the new menu entry
+		menu.addMenuEntry({
+			id: 'rich-workspace-init',
+			displayName: t('text', 'Add description'),
+			templateName: t('text', 'Readme') + '.' + loadState('text', 'default_file_extension'),
+			iconClass: 'icon-filetype-text',
```

Comment:
```suggestion
			iconClass: 'icon-rename',
```

To match the design mockup and have distinguishable icons:

![Screenshot 2022-12-08 at 14 27 06](https://user-images.githubusercontent.com/3404133/206458464-513cd984-ea18-4b1b-8068-c1d65011e1fb.png)

![](https://user-images.githubusercontent.com/925062/189093430-0a54fcda-63ed-4f8a-b89d-86d5cfe98508.png)